Makefile

```diff
@@ -142,3 +140,50 @@ CLIENT_GEN=$(shell go env GOPATH)/bin/client-gen
 else
 CLIENT_GEN=$(shell which client-gen)
 endif
+
+.PHONY: sbom
+sbom: spdx-generator
+	mkdir -p sbom
+	$(SPDX_GENERATOR) -o ./sbom
+
+.PHONY: spdx-generator
+spdx-generator:
+ifeq (, $(shell which spdx-sbom-generator))
+	mkdir -p sbom
+	curl -L https://github.com/spdx/spdx-sbom-generator/releases/download/v0.0.10/spdx-sbom-generator-v0.0.10-linux-amd64.tar.gz -o ./sbom/spdx-sbom-generator-v0.0.10-linux-amd64.tar.gz
+	tar xzvf ./sbom/spdx-sbom-generator-v0.0.10-linux-amd64.tar.gz -C sbom
+SPDX_GENERATOR=./sbom/spdx-sbom-generator
+else
+SPDX_GENERATOR=$(shell which spdx-sbom-generator)
+endif
+
+.PHONY: release
+release:
+	rm -rf release
+	mkdir -p ./release
+
+	# Build the kubectl plugins
+
+	GOOS=linux GOARCH=amd64 make bin/kubectl-schemahero
```

Comment:
IMO we should add `rm bin/kubectl-schemahero` before each of these - while currently it'll work because `bin/kubectl-schemahero` is marked as phony, that's really not best practice for makefiles